src/brandedVoice/BrandedVoice.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { TextBrand } from '../typography/brand'
+import { TextSubHeader } from '../typography/subHeader'
+import { StyledBrandedVoice, StyledContent, StyledIcon, StyledWrapper } from './BrandedVoice.style'
+
+// Should we consider having icon sizes in branding?
+const BRANDED_VOICE_ICON_SIZE = 40
```

Comment:
Does this value relate to title value or Icon, Profile sizing? 
Maybe we need to expose it as you suggested. So I go to creating a variable in branding, yes

Reply:
Avatar and icons share the same sizes? If so, i'll create a ticket to put these in branding